ui/js/directives/treeherder/clonejobs.js

```diff
@@ -911,24 +919,23 @@ treeherder.directive('thCloneJobs', [
                     resultset.platforms[j].name,
                     resultset.platforms[j].option
                 );
-
-                row = $('#' + platformId);
-
-                if($(row).prop('tagName') !== 'TR'){
-                    // First time the row is being created
-                    row = $('<tr></tr>');
-                    row.prop('id', platformId);
-                } else {
-                    // Clear and re-write the div content if it
-                    // already exists
-                    $(row).empty();
-                }
+                // We first determine whether the row has some visible element
+                var display_style = "none";
+                resultset.platforms[j].groups.forEach(function(group) {
+                    group.jobs.forEach(function(job) {
+                        if (filterWithRunnable(job)) {
+                            display_style = "table-row";
+                        }
+                    });
+                });
+                var rowHtml = "";
+                rowHtml += '<tr id="' + platformId + '" style="display: ' + display_style + ';">';
```

Comment:
You could just saw `rowHTML = '<tr id="' + platformId + '" style="display: ' + display_style + ';">';`, instead of reassigning it here.